gr-qtgui/python/qtgui/bindings/CMakeLists.txt

```diff
@@ -36,15 +34,12 @@ list(
     # plot_raster_python.cc
     # plot_waterfall_python.cc
     qtgui_types_python.cc
-    sink_c_python.cc
-    sink_f_python.cc
+    sink_python.cc
     # spectrumUpdateEvents_python.cc
     # spectrumdisplayform_python.cc
     # timeRasterGlobalData_python.cc
```

Comment:
I don't think we'll ever expose these to python. Might be a good time to remove.